docs/introduction/faq.md

```diff
@@ -0,0 +1,216 @@
+
+‘Warm' storage can be done with a small amount of trust, where you make a deal with a miner with a start date quite far in the future. The miner can choose to store your data in a sector now (but they won't get paid for proving it until the actual start date) or they can hold it for you (and even send you proofs of it on request) and you can then send them new data to overwrite it, along with a new storage deal that overwrites the previous one.
+
+There’s a pretty large design space here, and we can do a bunch of different things depending on the levels of trust involved, the price sensitivity, and the frequency of updates clients desire.
+
+### What are the standards and conditions of the verifier, is there a selected institution to verify, can you tell us about this situation?
+
+There will be several different institutions. We don’t have more updates about Verfiers at this time. We will have more around mid July.
+
+### Will the existence of the Filecoin pool lead to the deviation of centralized storage from the vision of distributed storage?
+
+No – Filecoin creates a decentralized storage network in part by massively decreasing the barrier to entry to becoming a storage provider. Even if there were some large pools, anyone can join the network and provide storage with just a modest hardware purchase, and we expect clients to store their files with many diverse miners.
+
+Also, note that world location matters for mining: many clients will prefer miners in specific regions of the world, so this enables lots of miners to succeed across the world, where there is storage demand.
+
+### How can someone get an audit on global/market-wide pricing quickly enough to make a decision if nearby nodes could "crowd out" smaller and less-reachable nodes with higher prices?
+
+Miner pricing is communicated via signed 'storage asks'. Currently, the software just queries a miner directly for their asks, but that's not necessary. An aggregator service can query asks from a large number of miners, and present those quickly to clients, so clients can ask an aggregator for the best set of miners. Since the asks are signed, and have temporal validity information baked in, clients can trust asks received from anyone.
+
+### Even though Filecoin will be backed up to our normal pinning layer, we still need to know how quickly we can access data from the Filecoin network. How fast will retrieval be?
+
+Our latest tests show that a sector that holds data takes ~1 hour to unseal. 1-5 hours is our best real world estimate.
+
+We are also building a feature that will be available by launch to help with this. When proposing a deal to a storage miner, you can set a flag indicating that you want an extra, unsealed copy of your data to store, making retrieval faster.
```

Comment:
Replace this entire answer with:

If you are retrieving your data from IPFS or a remote pinning layer such as an [FPS](../build/core-products/filecoin-backed-pinning-services/), retrieval should take on the order of milliseconds to seconds in the worst case. Our latest tests for retrieval from the Filecoin network directly show that a sealed sector holding data takes ~1 hour to unseal. 1-5 hours is our best real world estimate to go from sector unsealing to ultimate delivery of the data. If you need faster data retrieval for your application, we recommend building on Powergate or an FPS.